classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,1215 @@
+/**
+	String body = '';
+    String x = '\n\tpublic void method(){'
+            +'\n\t\tInteger i = 0;'
+            +'\n\t\ti = 1;'
+            +'\n\t\ti=2;'
+            +'\n\t\ti=3;'
+        +'\n\t}';
+    for(Integer i = 0; i < 200;i++){
+        body+=x.replace('method','method'+i);
+    }
+    system.debug('>>>\n\n'+body);
+ */
+public class ClaytonTestClass2 {
+    public void method0(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method1(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method2(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method3(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method4(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method5(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method6(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method7(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method8(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method9(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method10(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method11(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method12(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method13(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method14(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method15(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method16(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method17(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method18(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method19(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method20(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method21(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method22(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method23(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method24(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method25(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method26(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method27(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method28(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method29(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method30(){
+		Integer i = 0;
```

Comment:
This variable doesn't seem to have a very meaningful name. I suggest to give this variable a concise but meaningful name that suggest its purpose.